example.rb

```diff
@@ -36,6 +37,8 @@
 
 puts "Writing the JSON data"
 
+puts JSON.pretty_generate(recording.as_json)
```

Comment:
Looks like this debug output got left in.

Reply:
I left it so that we can see right away the result. The data.json file contains unformatted JSON that we have to run through a formatter.

Reply:
I don't want this turned on in the production deployment, and it also causes a bunch of duplicate work since it's generating the json twice.

Reply:
This file example.rb isn't used in prod. Only in development and in the command line.

Reply:
Ah, I missed that. Should be fine then.